exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/vendors/ForUpdateOption.kt

```diff
@@ -9,26 +9,26 @@ import org.jetbrains.exposed.sql.Table
  */
 sealed class ForUpdateOption(open val querySuffix: String) {
 
-    internal object NoForUpdateOption : ForUpdateOption("") {
+    internal data object NoForUpdateOption : ForUpdateOption("") {
         override val querySuffix: String get() = error("querySuffix should not be called for NoForUpdateOption object")
     }
 
     /** Common clause that locks the rows retrieved by a SELECT statement against concurrent updates. */
-    object ForUpdate : ForUpdateOption("FOR UPDATE")
+    data object ForUpdate : ForUpdateOption("FOR UPDATE")
```

Comment:
@e5l Changes in this file are based on suggestion that 'Sealed sub-object can be converted to data object'. Given that it is an API change, would it be better to not include it in this PR? Or should I add it as a small note to the breaking change docs?

Reply:
let's include it